lib/components/DoxenCodeBox.vue

```diff
@@ -54,7 +55,7 @@ export default {
       default: true
     },
     code: {
-      type: String,
+      type: [String, Object],
```

Comment:
```suggestion
      type: null,
```

Reply:
I’ve also updated the `code` prop type to `null` so it can accept any input. 
Thanks for the feedback!

Reply:
I’ve updated the `code` prop type to `null` so it can accept any input. 
Thanks for the feedback!